vignettes/costofliving.Rmd

```diff
@@ -79,46 +90,45 @@ us_state_local_tax_rate
 
 ### Examples of Function Usage
 
-- This is an example of calculating the gross salary for a single person earning minimum wage in California:
+What is the gross salary for a single person earning minimum wage in California?
```

Comment:
If you don't want to do "-" at the beginning of the example, I think it's still helpful to label the different examples

e.g. Do an #### Example 1 here, then the same afterwards

Reply:
got it!

Reply:
Do you think it's okay if I ask two questions and label it together as example 7 (an example of the function usage of avg_col_multiple_states_plot_all()). 

So, both of these questions can be answered using the same one plot.

#### Example 7. What is the difference in average cost of living and net salary of a single person earning minimum wage across all states in the U.S.? What are the top five states that are livable to earn a minimum wage salary in?**

or would you prefer to separate it into example 7, and example 8 and then show the same plot

Reply:
I think they can be under the same example because they are addressing the same plot

Also, I think it's safe to say what those 5 states are from the plot? Because this is a plot for all states

Reply:
I was thinking that the user could look at the plot for the answer for what those states are

It was an example of a question that someone may ask and why they would use that plot function

Should I still list those "top five states" underneath the plot to answer the question?
I was thinking like since one example is what is the gross salary in California that printing gross_salary("CA") is the answer itself, and we didn't need to also type underneath: the gross salary in California in 2022 is $_.

Reply:
That's fine then

Reply:
And TBH even the top five states that are "livable" in are all in the negatives

Reply:
unspoken conclusion: 
none of the states are livable earning a minimum wage salary ... :(

Reply:
Lol rip do you actually think of including that? I would say probably not tho

Reply:
lol no xDDDD